platform/saivpp/vpplib/SwitchStateBaseFdb.cpp

```diff
@@ -1082,3 +1082,352 @@ sai_status_t SwitchStateBase::vpp_delete_bvi_interface(
+
+    return SAI_STATUS_SUCCESS;
+
+}
+
+sai_status_t SwitchStateBase::vpp_fdbentry_add(
+        _In_ const std::string &serializedObjectId,
+        _In_ sai_object_id_t switch_id,
+        _In_ uint32_t attr_count,
+        _In_ const sai_attribute_t *attr_list)
+{
+
+    SWSS_LOG_ENTER();
+             
+    sai_fdb_entry_t fdb_entry;
+    sai_deserialize_fdb_entry(serializedObjectId, fdb_entry);
+
+    /* Attribute#1 */
+    auto attr_type = sai_metadata_get_attr_by_id(SAI_FDB_ENTRY_ATTR_TYPE, attr_count, attr_list);
+    if (attr_type == NULL)
+    {
+        SWSS_LOG_ERROR("attr SAI_FDB_ENTRY_ATTR_TYPE was not passed");
+
+        return SAI_STATUS_FAILURE;
+    }
+    bool is_static = (attr_type->value.s32 == SAI_FDB_ENTRY_TYPE_STATIC ? true : false);
+    bool is_add = true; /* Adding the entry in FDB*/
+
+    /* Attribute#2 */  
+    sai_object_id_t br_port_id;
+    sai_object_id_t port_id;
+
+    attr_type = sai_metadata_get_attr_by_id(SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID, attr_count, attr_list);
+    if (attr_type == NULL)
+    {
+        SWSS_LOG_ERROR("attr SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID was not passed");
+
+        return SAI_STATUS_FAILURE;
+    }
+    br_port_id = attr_type->value.oid;
+    sai_object_type_t obj_type = objectTypeQuery(br_port_id);
+
+    if (obj_type != SAI_OBJECT_TYPE_BRIDGE_PORT)
+    {
+        SWSS_LOG_ERROR("SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID=%s expected to be PORT but is: %s",
+                sai_serialize_object_id(br_port_id).c_str(),
+                sai_serialize_object_type(obj_type).c_str());
+
+        return SAI_STATUS_FAILURE;
+    }
+
+    auto br_port_attrs = m_objectHash.at(SAI_OBJECT_TYPE_BRIDGE_PORT).at(sai_serialize_object_id(br_port_id));
+    auto meta = sai_metadata_get_attr_metadata(SAI_OBJECT_TYPE_BRIDGE_PORT, SAI_BRIDGE_PORT_ATTR_PORT_ID);
+    auto bp_attr = br_port_attrs[meta->attridname];
+    port_id = bp_attr->getAttr()->value.oid;
+    obj_type = objectTypeQuery(port_id);
+
+    if (obj_type != SAI_OBJECT_TYPE_PORT)
+    {
+        SWSS_LOG_NOTICE("SAI_BRIDGE_PORT_ATTR_PORT_ID=%s expected to be PORT but is: %s",
+                sai_serialize_object_id(port_id).c_str(),
+                sai_serialize_object_type(obj_type).c_str());
+        return SAI_STATUS_FAILURE;
+    }
+
+    /* Need to extract the VLAN ID attached based on the Port_ID */
+    sai_attribute_t attr;
+    attr.id = SAI_PORT_ATTR_PORT_VLAN_ID;
+
+    sai_status_t get_status = get(SAI_OBJECT_TYPE_PORT, port_id, 1, &attr);
+
+    if (get_status != SAI_STATUS_SUCCESS)
+    {
+        SWSS_LOG_WARN("failed to get port vlan id from port %s", sai_serialize_object_id(port_id).c_str());
+        return SAI_STATUS_FAILURE;
+    }
+
+    uint32_t bd_id = attr.value.u16; /* bd_id is same as VLAN ID for .1Q bridge */
+
+    std::string ifname;
+    if (vpp_get_hwif_name(port_id, 0, ifname) == true) 
+    {
+        const char *hwif_name = ifname.c_str();
+        SWSS_LOG_NOTICE(" FDB_ENTRY on hwif_name %s ", hwif_name);
```

Comment:
Combine both SWSS_LOG_NOTICE into one with ret value success or fail for given hwif_name. No need to print two messages.